anomaly_detector/exception/exceptions.py

```diff
@@ -39,3 +39,11 @@ class EmptyDataSetException(Exception):
     def __init__(self, msg="Empty dataset returned from ingest"):
         """Initialize message."""
         self.message = msg
+
+
+class InvalidStorageProxyConfigException(Exception):
+    """Validates there is empty dataset returned from ingest."""
```

Comment:
Typos here, suggestion on rewording: Validate that there was an empty dataset returned from ingest.

Reply:
I've removed this internally.